### PR TITLE
CANS-901: When user navigates away from a view-only assessment, the navigation warning message will NOT appear

### PR DESCRIPTION
**Pull request by DmitriyRudenko**
## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by DmitriyRudenko** [comment]
Jenkins retest this please

File: app/javascript/Application/components/Assessment/AssessmentContainer.js [hunk 1]
```diff
@@ -79,6 +78,9 @@ class AssessmentContainer extends Component {
     const assessment = this.state.assessment
     const isEditable = Boolean(!assessment || !assessment.id || isAuthorized(assessment, 'update'))
     this.setState({ isEditable })
+    if (isEditable) {
+      window.addEventListener('beforeunload', alertMessage)
```
Comment (nasar-cambria) [inline app/javascript/Application/components/Assessment/AssessmentContainer.js]: beforeunload are two words. Should this be camel Casing?

Comment (DmitriyRudenko) [reply app/javascript/Application/components/Assessment/AssessmentContainer.js]: Nope: https://developer.mozilla.org/en-US/docs/Web/Events/beforeunload

**Comment by LeonidMarushevskyi** [comment]
Retest this please

**Comment by LeonidMarushevskyi** [comment]
Retest this please

**Pull request merged**